content/2-3.md

```diff
@@ -67,58 +70,59 @@ Skills required for this module.
+  * [HTML `longdesc` attribute {% include_cached icon.html name="external-link" %}](https://html.spec.whatwg.org/multipage/obsolete.html#dom-frame-longdesc)
+  * [HTML `picture` element{% include_cached icon.html name="external-link" %}](https://html.spec.whatwg.org/multipage/embedded-content.html#the-picture-element)
+  * [HTML `Figure` element {% include_cached icon.html name="external-link" %}](https://html.spec.whatwg.org/multipage/grouping-content.html#the-figure-element)
+  * [HTML `figcaption` element {% include_cached icon.html name="external-link" %}](https://html.spec.whatwg.org/multipage/grouping-content.html#the-figcaption-element)
   * [CSS Specifications](https://www.w3.org/TR/CSS)
-  * [ARIA `img`](https://www.w3.org/TR/wai-aria-1.1/#img) role
-  * WAI-ARIA attributes [`aria-label`](https://www.w3.org/TR/wai-aria-1.1/#aria-label), [`aria-labelledby`](https://www.w3.org/TR/wai-aria-1.1/#aria-labelledby), and [`aria-describedby`](https://www.w3.org/TR/wai-aria-1.1/#aria-describedby)
-  * [`Figure`](https://html.spec.whatwg.org/multipage/grouping-content.html#the-figure-element) and [`figcaption`](https://html.spec.whatwg.org/multipage/grouping-content.html#the-figcaption-element) elements
-  * In-depth knowledge of [Prerequisites for Students](/curricula/developing-accessible-content/#prerequisites-for-students)
+  * [SVG](https://www.w3.org/TR/SVG2/)
+  * [WAI-ARIA `img` role](https://www.w3.org/TR/wai-aria-1.1/#img)
+  * [WAI-ARIA `aria-label` attribute](https://www.w3.org/TR/wai-aria-1.1/#aria-label)
+  * [WAI-ARIA `aria-labelledby` attribute](https://www.w3.org/TR/wai-aria-1.1/#aria-labelledby), 
+  * [WAI-ARIA `aria-describedby` attribute](https://www.w3.org/TR/wai-aria-1.1/#aria-describedby)
+* In-depth knowledge of [Prerequisites for Students](/curricula/developer-modules/#prerequisites-for-students)
 
 {% include excol.html type="end" %}
 
 ## Topics to Teach
 
-Optional topics to achieve the learning outcomes.
+Topics to support the teaching sequence:
 
 {% include excol.html type="start" %}
 
-### Topic: Simple Images
+### Topic: Text Alternatives
 
 {% include excol.html type="middle" %}
 
-Explain that simple images and graphics, such as informative, textual, and decorative, only require a short text alternative to describe them. Mention scenarios where text alternatives allow people to access information which would not be available otherwise. Relate these strategies to WCAG success criterion [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/quickref/#non-text-content).
+Demonstrate how text alternatives allow people to access information encoded in images. Explain how text provides a base format that can be presented in different ways, for example visually in different text sizes, through text-to-speech, as braille, and as symbols. Explain different coding techniques to provide text alternatives for images.
```

Comment:
"information included in images" instead of "information encoded in images"?

Reply:
Good suggestion. It is there because of the technical implications of "encode" information using an image format, but probably "include" is easier to understand.